schema.sql

```diff
@@ -52,4 +52,25 @@ CREATE TABLE visits(
 	animal_id INTEGER REFERENCES animals(id),
 	vet_id INTEGER REFERENCES vets(id),
 	date_of_visit DATE
-)
\ No newline at end of file
+)
+
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍